desc/compute/_geometry.py

```diff
@@ -196,15 +196,19 @@ def _R0_over_a(params, transforms, profiles, data, **kwargs):
     data=["sqrt(g)", "g_tt"],
 )
 def _a_major_over_a_minor(params, transforms, profiles, data, **kwargs):
-    P = compress(  # perimeter
-        transforms["grid"],
-        surface_integrals(
+    max_rho = transforms["grid"].nodes[transforms["grid"].unique_rho_idx[-1], 0]
+    P = (
+        compress(  # perimeter
             transforms["grid"],
-            jnp.sqrt(data["g_tt"]),
+            line_integrals(
+                transforms["grid"],
+                jnp.sqrt(data["g_tt"]),
+                line_label="theta",
+                fix_surface=("rho", max_rho),
+            ),
             surface_label="zeta",
-            max_surface=True,
-        ),
-        surface_label="zeta",
+        )
+        / max_rho
```

Comment:
Should this be multiplied by `max_rho` instead?

Reply:
i think this is correct. P ~ r, if we compute P0 at r=r0 but want the value at r=1, thats P1 = P0*1/r0